### PR TITLE
Ensure things compile when F_GET_SEALS is defined

### PR DESCRIPTION
**Pull request by k0001**

### CODE REVIEW

File: src/System/Posix/FileControl.hsc [hunk 1]
```diff
@@ -487,11 +487,11 @@ newtype DNotify = DNotify CInt
 
 #if defined(F_GET_SEALS)
 
-newtype FileSeal = FileSeal CInt
+newtype Seal = Seal CInt
```
Comment (maoe) [inline src/System/Posix/FileControl.hsc]: Could we rename this to `Seals`?

**Review by maoe — CHANGES_REQUESTED** [review]
Thanks!

BTW `F_GET_SEALS` isn't available on my systems. What OS are you using?

**Comment by maoe** [comment]
On Ubuntu F_GET_SEALS seems to be available in linux/fcntl.h but not in fcntl.h.

**Comment by k0001** [comment]
Thanks for the quick reply.

I renamed `Seal` to `Seals`.

I am trying to build this on NixOS.

**Comment by maoe** [comment]
Looks good. Thank you!

Regarding the header files and the availability of the macro, I’ll file a separate ticket.

**Pull request merged**